src/main/java/bio/terra/app/usermetrics/BardClient.java

```diff
@@ -41,13 +40,12 @@ public class BardClient {
   private static final String SYNC_PATH = "/api/syncProfile";
 
   @Autowired
-  public BardClient(UserMetricsConfiguration metricsConfig) {
-    this.restTemplate = new RestTemplate();
-    restTemplate.setRequestFactory(new HttpComponentsClientHttpRequestFactory());
+  public BardClient(UserMetricsConfiguration metricsConfig, RestTemplate restTemplate) {
+    this.restTemplate = restTemplate;
 
     this.headers = new HttpHeaders();
     headers.setContentType(MediaType.APPLICATION_JSON);
-    headers.setAccept(Arrays.asList(MediaType.APPLICATION_JSON, MediaType.APPLICATION_JSON));
```

Comment:
I assumed that these duplicate headers were an oversight.

Reply:
Yeah, that seems like a bug